pkg/mab/reward.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2015 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package mab
+
+type Reward struct {
+	Count      int
+	TotalCov   float64 // sum(cov)
```

Comment:
Is it possible to avoid use of floats entirely? They tend to create more problems than solve. Sometimes tests end up being non-deterministic and produce failure flakes and coverage flakes on CI.